src/macros.rs

```diff
@@ -0,0 +1,17 @@
+#[macro_export]
+macro_rules! exit {
+    ($ec:expr, $($message:expr), +) => {
+        {eprintln!($($message), +);
+        std::process::exit($ec);}
+    };
+}
+
+#[macro_export]
+macro_rules! matches_any {
+    ($value:expr, $first:expr) => {
+        $value.eq_ignore_ascii_case($first)
+    };
+    ($value:expr, $first:expr, $($pattern:expr), +) => {
+        $value.eq_ignore_ascii_case($first) || matches_any!($value, $($pattern), +)
```

Comment:
eq_ignore_ascii_case is from the unstable `osstring_ascii`

Reply:
Oh, i just found it there, and used it: https://doc.rust-lang.org/std/primitive.str.html#method.eq_ignore_ascii_case

Reply:
What would you recommend?

Reply:
The problem here is that I was not able to build with latest nightly, beta or stable. I think we can use this no problem especially since it is stable in nightly but as long as we need a specific compiler version (latest nightly does currently not work cause it breaks twitch_api2) we should note which version works somewhere (ie. put the exact version in README or something like that).

Reply:
@cafce25 please try again.
cargo 1.53.0-nightly now works